aea/skills/error/handlers.py

```diff
@@ -67,10 +68,13 @@ def send_unsupported_protocol(self, envelope: Envelope) -> None:
         """
         logger.warning("Unsupported protocol: {}".format(envelope.protocol_id))
         reply = DefaultMessage(
-            type=DefaultMessage.Type.ERROR,
-            error_code=DefaultMessage.ErrorCode.UNSUPPORTED_PROTOCOL,
+            dialogue_reference=("", ""),
+            message_id=1,
+            target=0,
+            performative=DefaultMessage.Performative.ERROR,
+            error_code=ErrorCode.UNSUPPORTED_PROTOCOL,
```

Comment:
Note, @marcofavorito since ErrorCode is not defined fully by the specification we had to move it into its own file (to be able to completely generate `message.py`). What do you think of the current implementation?

Reply:
That is fine.